packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -172,8 +188,129 @@ kind: ConfigMap
 metadata:
   name: {{ template "common.fullname" . }}-geth-config
   labels:
-{{ include "standard.labels" .  | indent 4 }}
+{{ include "common.standard.labels" .  | indent 4 }}
 data:
   networkid: {{ .Values.genesis.networkId | quote }}
-  genesis.json: {{ .Values.genesis.fileBase64 | b64dec | quote }}
+  genesis.json: {{ .Values.genesis.genesisFileBase64 | b64dec | quote }}
+{{- end -}}
+
+{{- define "common.celotool-container" -}}
```

Comment:
Nit: maybe a name that indicates this is generating values for a validator would make this easier to understand